staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -182,10 +182,17 @@ func finishRequest(timeout time.Duration, fn resultFunc) (result runtime.Object,
 	panicCh := make(chan interface{}, 1)
 	go func() {
 		// panics don't cross goroutine boundaries, so we have to handle ourselves
-		defer utilruntime.HandleCrash(func(panicReason interface{}) {
+		defer func() {
+			panicReason := recover()
+			if panicReason != nil {
+				const size = 64 << 10
+				buf := make([]byte, size)
+				buf = buf[:goruntime.Stack(buf, false)]
+				panicReason = fmt.Sprintf("%v\n%s", panicReason, string(buf))
```

Comment:
trim trailing newline

Reply:
Makes concatenation of further backtraces cleaner

Reply:
done